### ISSUE TITLE
Promoting Rchain on different large group on social media

### ISSUE THREAD

**Issue by philipandri002** [rchain/bounties#656]
*promoting Rchain on several group's especially in Africa will create a stronger awareness and confident in the large and growing population of Africa. For example I know some social media group designed for entrepreneurs. Such a platform would give young Africa entrepreneurs the opportunity to look at this great opportunity that Rchain offers.
Don't forget Nigerians especially have a very keen interest in the use and development of cryptocurrency* 

[Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.]

*Estimated Budget of Task:* **$[500]**  \
*Estimated Timeline Required to Complete the Task:* **[2weeks]**  \
*Measure of Completion:* [#Done]
  - [x] adding to the RChain platform a minimum of 20-50 new persons

[See CONTRIBUTING.md for details on budget and reward process.]

--  \
***Legal***  \
*Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.*

**Comment by David405** [comment]
@philipandri002 kindly make this issue more precise or S.M.A.R.T., For instance, what social media platforms are you referring to? what are the specific tasks to be done?

**Comment by casanwugo** [comment]
@philipandri002, What you've got here is an interesting topic. It is broad, and cuts across sensitive areas like social media and marketing. 

To be more effective at building relationships with new members online, RChain need a cross-functional social media team, one where marketing works together with other departments.

Distributing social responsibilities to relevant people across the organization can be efficient, be effective, and help make one-on-one member engagement scalable

**Comment by jasoncruzzy** [comment]
@philipandri002  Nice idea,  this will definitely expand and awareness for our community

**Comment by azazime** [comment]
Quite interesting topic, Now firstly I must say Nigerians/Africans are very active on social media so it would be a less difficult achieving our aim. I would like to suggest we run a sponsored AD on Instagram (I would prefer Instagram because at the moment its trending more than other social media platforms). The AD on Rchain should be sent to a targeted audience of young entrepreneurs at least twice a day for the next two weeks. That would go a long way in creating awareness of Rchain within the instagram community

**Comment by awesome000** [comment]
Twitter as we all know is a global community, most of the things that trend and go viral starts from Twitter, and Africans have not been exempted on its benefits. Most of us here are on that social platform for different reasons and we could make use of the number we have. We could go as far as putting up a token as a reward for anybody who tweets about the importance of Rchain and get the highest retweet. By so doing we reach out to the African community more and impact more on lives. They are people who specifically run promotion on Twitter, we can also reach out to them tell them the benefit of Rchain and convince them to tweet about it, or better still pay for there services and put Rchain up for the Africa to see.

**Comment by casanwugo** [comment]
May I contribute to this by adding that Facebook will be a quick win to a large extent. It's users are grossly involved in spending quality time on the net, and they cut across different category of people. 

RChain is the brainchild of Greg Meredith, a mathematician and software engineer who has previously worked at Microsoft. The Board and Cooperative includes Vlad Zamfir, lead researcher for the Ethereum Foundation’s Casper Proof of Stake protocol, Alexandr Bulkin co-founder of Coinfund, Kenny Rowe co-founder of MakerDAO, and Ethereum developer, David Currin, who is working with MakerDAO and DappHub.

Reflective states that it will enable startup teams to connect with RChain senior leadership, as well as talent from Amazon, Facebook, Google, IBM, and Microsoft.

**Comment by philipandri002** [comment]
Great! You see there are lots of talent especially in the young people of Africa which have not been honestly given the opportunity to express these talent.. RChain could possibly be that platform and also enjoy the benefits of these fresh talent..

Like someone said Africa is a fresh and fertile ground to take advantage of

**Comment by jasoncruzzy** [comment]
@dckc I think its a bit different because the channels and moderators we have are groups created specifically for Rchain related issues but @philipandri002 meant we publicize Rchain on already existing 'non-Rchain' groups on various social media platforms  to create awareness of Rchain to the 'non-Rchain' group  members. @dckc hopefully you get the point now.

**Comment by dckc** [comment]
OK, I see the difference.

What does it look like when it's done? What's your measure of success?

**Comment by philipandri002** [comment]
It would have created a level of awareness about Rchain and it's potential of fixing block chain scalability and timing

**Comment by dckc** [comment]
What level of awareness? a SMART objective includes an objective measure of success.

**Comment by dckc** [comment]
Yes, that's clear. To make it easy to find, I copied the 20-50 measure to the top of the issue.

**Comment by philipandri002** [comment]
@pmooreman @dckc @kitblake

**Comment by philipandri002** [comment]
I started reaching out to most who made some inquiry from me to see how far they went with the registration

**Comment by philipandri002** [comment]
![screenshot_20180607-134826](https://user-images.githubusercontent.com/38182897/41100575-acd61528-6a59-11e8-8228-75c6e7596318.png)
That's like what I am talking about

**Comment by kitblake** [comment]
Imho we shouldn't have bounties for talking about RChain on Facebook and WhatsApp. 

Granted, doing so does raise awareness about RChain. The lady in the screenshot might decide to invest. But new investors is not what RChain is looking for. RChain is looking for new developers to come onboard. That is a verifiable action, just ask the new member to mention @philipandri002 when they meet with the Greeter. 

Which reminds me, @philipandri002, are you a Member? You're not verified on Discord. Maybe you're waiting to see if you get paid before you pay the $20, which is understandable. 

Afaik none of the "adding to the RChain platform a minimum of 20-50 new persons" have come on board. But the intention is good and it could happen in the course of June so let's see.

**Comment by AyAyRon-P** [comment]
Several things should happen to claim a bounty on this:

- Please provide links and screenshots of groups related to and regarding promotion of RChain. 
- Identify how many people have seen/viewed and become active from this posting. 
- Determine a real measure of completion, not just "DONE" 
- What is the real ROI that RChain will gain from this? How will these promotions help spread the search for new developers?

**Comment by philipandri002** [comment]
@AyAyRon-P okay I think I have answered lots of your questions in previous comment. But to round it up this campaign has successfully brought a new developer to the RChain and you will find him in discord as @MrBoss @Biller on github..

Thanks.
